packages/block-editor/src/store/selectors.js

```diff
@@ -599,6 +599,23 @@ export const getMultiSelectedBlockClientIds = createSelector(
 	],
 );
 
+/**
+ * Returns the current multi-selection set of block client IDs, or an empty
+ * array if there is no multi-selection.
+ *
+ * @param {Object} state Editor state.
+ *
+ * @return {Array} Multi-selected block client IDs.
+ */
+export function getMultiSelectedBlockClientIds( state ) {
+	const { start, end } = state.blockSelection;
+	if ( start === end ) {
+		return EMPTY_ARRAY;
+	}
+
+	return getSelectedBlockClientIds( state );
```

Comment:
If we wanted to push more on the abstraction, could it be fair to say that if the result of `getSelectedBlockClientIds` is `.length <= 1`, then it'd fair to return `EMPTY_ARRAY` ? Rather than referencing raw values in the condition before this.

Reply:
Yes, but I suspect that avoid calling `getSelectedBlockClientIds` is way more performant as there's no need for memoization...